phobos/defs.py

```diff
@@ -4,6 +4,9 @@
 RPY_CONVENTION = 'xyz'
 
 HYRODYN_AVAILABLE = False
+BASE_LOG_LEVEL = "INFO"
+LOG_FILE_CONVENTION = "cli_phobos.log"
+
```

Comment:
We should set the default here to None and write the log only to a file if specified.